hadoop-hdds/container-service/src/main/java/org/apache/hadoop/ozone/container/keyvalue/statemachine/background/BlockDeletingService.java

```diff
@@ -601,4 +603,32 @@ void apply(Table<?, DeletedBlocksTransaction> deleteTxnsTable,
   public BlockDeletingServiceMetrics getMetrics() {
     return metrics;
   }
+
+  /**
+   * The wrapper class of the result of deleting transactions.
+   */
+  private class DeleteTransactionStats {
```

Comment:
This class should be static.

```suggestion
  private static class DeleteTransactionStats {
```